simcon_project/conversation_templates/views/conversation.py

```diff
@@ -0,0 +1,140 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponseNotFound
+from django.forms import modelformset_factory
+from conversation_templates.models.conv_template import ConversationTemplate
+from conversation_templates.models.template_node import TemplateNode
+from conversation_templates.models.template_node_response import TemplateNodeResponse
+from conversation_templates.models.template_response import TemplateResponse
+from conversation_templates.forms import TemplateNodeChoiceForm
+from users.models.student import Student
+from users.models.assignment import Assignment
+from datetime import datetime
+from django.contrib.auth.decorators import login_required
+import django_tables2 as tables
+
+# Globals
+ct_templates_dir = 'conversation'
+
+
+class NodeDescriptionTable(tables.Table):
+    """
+    Creates table that displays the description for each TemplateNode object that a Student visited.
+    """
+    node_description = tables.Column()
+
+
+# Views
+@login_required(login_url='/accounts/login/')
```

Comment:
My understanding is that this redirects to the login page if the user isn't logged in already?

Reply:
Yes